.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2017,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "env": {
+    "browser": true,
+    "node": true,
+    "es6": true
+  },
+  "plugins": ["prettier"],
```

Comment:
When using recommended,  `"plugins": ["prettier"],` can be removed as it's already [part of recommended](https://github.com/prettier/eslint-plugin-prettier/blob/master/README.md#recommended-configuration).

Reply:
Oh, you're right. Missed that 👍